app/scripts/proactive/view/dom.js

```diff
@@ -1027,16 +1027,24 @@ define(
 
             $(document).keydown(function (e) {
                 if (ctrlDown && e.keyCode == cKey) {
+                    task = [];
                     console.log("copy");
                     copied = [];
                     $(".selected-task").each(function (i, t) {
                         copied.push(t);
+                        task.push($(t).data( "view" ))
                     })
                 }
                 if (ctrlDown && e.keyCode == vKey) {
                     if (pasteAllow) {
-                        console.log("paste");
-                        require('StudioApp').views.workflowView.copyPasteTasks(copied, pasteAllow);
+                        var newTaskModel = []
+                        var Tasksview = [];
+                         $.each(copied, function (i, t) {
```

Comment:
As you loop in copied to get elements for task, maybe you don't need copied table anymore. Also, it seems that there are several tasks in 'task' so you might rename it 'tasks'